src/e3/anod/spec.py

```diff
@@ -54,9 +54,7 @@
     SOURCE_PRIMITIVE = Literal["source"]
 
     # Anod Dependency can target a build, install, or source
-    DEPENDENCY_PRIMITIVE = Union[
-        BUILD_PRIMITIVE, INSTALL_PRIMITIVE, SOURCE_PRIMITIVE,
-    ]
+    DEPENDENCY_PRIMITIVE = Union[BUILD_PRIMITIVE, INSTALL_PRIMITIVE, SOURCE_PRIMITIVE]
```

Comment:
why Union is still here?

Reply:
Because it's needed in such cases. That's not a type annotation here.